evasquez26/angular4-startup
# S2-ONE-47-55 See my courses and delete courses

## Story / Task

[ONE-55](https://bixlabs.atlassian.net/browse/ONE-55) and [ONE-47](https://bixlabs.atlassian.net/browse/ONE-47)


## What this PR does?

Adds the functionality to see a list of courses and to be able to delete a course.


### Before 

We could not see a list of courses

### After

- We can see a list of courses by taken or taking (student in studyGroup context that has or not finished), given or giving (teacher in studyGroup context) and created (admin in course context)
- If we create a course and go to the created list of courses then we can delete the course

### How to test / reproduce

- Login as a teacher
- Create a course
- You should see the course listed on the created courses menu option on the left

### TODOs

N/A

### Related PRs

N/A

## Migrations

NO
